test-e2e/tests/watch_history/logged_in_test.ts

```diff
@@ -10,41 +9,20 @@ runTestSuite(testConfigs.jwpAuth, testConfigs.jwpAuthNoWatchlist, 'JW Player');
 runTestSuite(testConfigs.cleengAuthvod, testConfigs.cleengAuthvodNoWatchlist, 'Cleeng');
 
 function runTestSuite(config: typeof testConfigs.svod, configNoWatchlist: typeof testConfigs.jwpAuthNoWatchlist, providerName: string) {
-  let loginContext: LoginContext;
-
   Feature(`watch_history - logged in - ${providerName}`).retry(Number(process.env.TEST_RETRY_COUNT) || 0);
 
   Before(({ I }) => {
     I.useConfig(config);
   });
 
-  Scenario(`I can get my watch history when logged in - ${providerName}`, async ({ I }) => {
-    await registerOrLogin(I);
-
-    // New user has no continue watching history shelf
-    I.dontSee(constants.continueWatchingShelfTitle);
-
-    await I.openVideoCard(videoTitle);
-
-    await playVideo(I, 0, videoTitle);
-    I.see(constants.startWatchingButton);
-    I.dontSee(constants.continueWatchingButton);
-
-    await playVideo(I, 80, videoTitle);
-
-    I.see(constants.continueWatchingButton);
-    I.dontSee(constants.startWatchingButton);
-    await checkProgress(I, `//button[contains(., "${constants.continueWatchingButton}")]`, (80 / videoLength) * 100, 5, '_progressRail_', '_progress_');
-  });
-
   Scenario(`I can get my watch history stored to my account after login - ${providerName}`, async ({ I }) => {
     I.dontSee(constants.continueWatchingShelfTitle);
 
     await I.openVideoCard(videoTitle);
     I.dontSee(constants.continueWatchingButton);
     I.see(constants.startWatchingButton);
 
-    await registerOrLogin(I);
+    await registerAndPlayVideo(I, videoTitle);
```

Comment:
This test has been changed subtly but significantly here. In the old version it tests that a user's saved watch history is restored after logging in on a fresh browser. This test only tests that it is saved within the same session. The restore isn't tested.